karrot/offers/tests/test_api.py

```diff
@@ -181,7 +181,8 @@ def test_create_offer(self, prepare_email):
                     'image': image_file
                 }],
             }
-            response = self.client.post('/api/offers/', data=encode_data_with_images(data))
+            with self.captureOnCommitCallbacks(execute=True):
```

Comment:
Django 3.2 allows us to use this instead of `TransactionTestCase`, should speed up test run!

Reply:
Nice! Seems a good find :+1: